lib/src/date/controller.dart

```diff
@@ -143,7 +143,7 @@ class DatePageValue {
   int get visibleDayCount => visibleRange.visibleDayCount;
 
   final double page;
-  DateTime get date => DateTimeTimetable.dateFromPage(page.floor());
+  DateTime get date => DateTimeTimetable.dateFromPage(page.round());
```

Comment:
I could also change this back, but in my eyes, this makes more sense to the user